src/triggers/GUILD_CREATE/birthdayScheduler.ts

```diff
@@ -0,0 +1,64 @@
+import cron from 'cron';
+import * as Discord from '../../Discord';
+import Trigger from '../../Trigger';
+import { Collection, Config, BirthdaysConfig } from '../../database';
+import { calculateDayOfYear } from '../../commands/birthday/_common';
+import { CreateMessage } from '../../Discord/message';
+
+const scheduler = new Trigger<Discord.Event.CHANNEL_UPDATE>({
+  event: Discord.Event.CHANNEL_UPDATE,
+  handler: async ctx => {
+    const guildId = ctx.getData().id;
+
+    const birthdaysCfg = await ctx.db.getConfigValue<BirthdaysConfig>(
+      guildId,
+      Config.BIRTHDAYS,
+    );
+
+    if (!birthdaysCfg) {
+      throw new Error('No birthdays configuration found');
+    }
+
+    const channel = await birthdaysCfg.channel;
+    const messages = await birthdaysCfg.messages;
+
+    if (!messages || messages.length === 0) {
+      throw new Error('No birthday messages configured');
+    }
+
+    // Run cron at 10:15 am every day.
+    const test = new cron.CronJob('00 15 10 * * *', async () => {
```

Comment:
Maybe make it so mods can configure the time of birthday notifications through a command?